component/src/test/java/org/wso2/extension/siddhi/execution/geo/GeocoordinateStreamFunctionProcessorTest.java

```diff
@@ -0,0 +1,157 @@
+/*
+ *  Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is this file committed in 2017?

Reply:
Fixed https://github.com/wso2-extensions/siddhi-execution-geo/pull/61/commits/a08d5e724a57f053d7d18ae441ace09bafeb208a